### PR TITLE
Add Cloud Functions example

### PR DESCRIPTION
**Pull request by kolodny**

### CODE REVIEW

File: node/cloud-functions/README.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    ----------------------------------------
+    hello-8
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "0"
+    ----------------------------------------
+    hello-9
+      cf1:isPrime                              @ 2017/11/03-14:35:32.533000
+        "0"
+
+## Cleaning up
```
Comment (sduskis) [inline node/cloud-functions/README.md]: Do the pub/sub topic and function have to be removed as well?

Comment (kolodny) [reply node/cloud-functions/README.md]: https://cloud.google.com/functions/pricing

It doesn't appear so, the pricing is a function (heh) of invocations and below 2 million has no costs.

File: node/cloud-functions/README.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+When deploying a Cloud Function, make sure you're in the directory that
+contains the `index.js` file you want to deploy:
+
+    cd cloud-functions
+
+Run the following command to deploy the Cloud Function (make sure to replace
+`BUCKET_NAME` with what you used for the bucket name):
+
+    gcloud beta functions deploy helloBigtable --stage-bucket BUCKET_NAME --trigger-topic hello_bigtable
```
Comment (sduskis) [inline node/cloud-functions/README.md]: Can you please add --region here, and specify that it should be in the same region as the Cloud Bigtable cluster?

Comment (kolodny) [reply node/cloud-functions/README.md]: Done

**Pull request merged**